frontend/src/components/pages/TeamDashboard/index.jsx

```diff
@@ -20,8 +22,10 @@ function DashboardUnconnected({
   let urlParams = useParams() || {};
 
   const teamId = parseInt(urlParams.teamId, 10);
-  const team = teams[teamId];
-  React.useEffect(() => {
+  const team = teams && teams[teamId];
+  activityLogDayCounts = [];
```

Comment:
So the way this was before was here

`activityLogDayCounts: Object.values(
      state.apiEntities.activityLogDayCounts || {}
    ),,`
    
Now it is just an empty array here. We need to retain the format it was inside mapStateToProps. There should be logic here to make sure we get the Object.values just like we did before.